ext/standard/image.c

```diff
@@ -1155,96 +1156,80 @@ static struct gfxinfo *php_handle_webp(php_stream * stream)
 }
 /* }}} */
 
-/* {{{ php_handle_avif
- * There's no simple way to get this information - so, for now, this is unsupported.
- * Simply return 0 for everything.
- */
-static struct gfxinfo *php_handle_avif(php_stream * stream) {
-	return ecalloc(1, sizeof(struct gfxinfo));
+/* {{{ User struct and stream read/skip implementations for libavifinfo API */
+struct php_avif_stream {
+    php_stream* stream;
+    uint8_t buffer[AVIFINFO_MAX_NUM_READ_BYTES];
+};
+
+static const uint8_t* php_avif_stream_read(void* stream, size_t num_bytes) {
+    struct php_avif_stream* avif_stream = (struct php_avif_stream*)stream;
+
+    if (avif_stream == NULL || avif_stream->stream == NULL)
+        return NULL;
```

Comment:
Please put curly brackets around the statement (here and elsewhere):
```suggestion
	if (avif_stream == NULL || avif_stream->stream == NULL) {
		return NULL;
	}
```

Reply:
Done.